src/config/nfs.md

```diff
@@ -0,0 +1,43 @@
+# NFS
+
+## Mounting an NFS Share
+
+To mount an NFS share, start by installing the `nfs-utils` package:
+
+```
+# xbps-install nfs-utils
+```
```

Comment:
From the contribution guide:

> Command code-blocks should not be used to describe routine tasks documented elsewhere in this Handbook. For example, when writing documentation for the foo package, do not provide a command code-block stating that one should install it via xbps-install foo. Similarly, do not provide code blocks describing how to enable the foo service.

You can start with an introdutory text right after the page title with content like "In order to mount or serve NFS shares, you should install the `nfs-utils` package.", since it's needed for both cases.